src/metabase/formatter.clj

```diff
@@ -265,22 +266,39 @@
              (format "%.8f° %s" (Math/abs v) dir)
              v)))))
 
+(defn dictionary-formatter
```

Comment:
```suggestion
(defn- dictionary-formatter
```

Since this isn't used outside of this namespace it might be wise to mark it private. That way if someone wants to tweak it later (add another parameter or something like that) they don't need to check and see if it's used elsewhere first.

Best to keep surface area of things small

Reply:
I'm also going to nitpick a little bit here and point out that _dictionary_, _map_, and _associative array_ are all basically synonyms in the computer science world but Clojure is in the _map_ camp so it probably be more idiomatic to use that name here instead of the alternatives.

```suggestion
(defn- map-formatter
```

Not that it makes a gigantic difference but not having to make that _dictionary => map_ translation in your head reduces the cognitive load of reading the code which makes it easier for other people to understand and work on going forward. If in doubt I like to error on the side of code that's easier to read

Reply:
Actually I guess we call it `:type/Dictionary` so this argument maybe doesn't make sense. I'm not even sure what to believe anymore.

Reply:
You know what forget I said anything about the name of this function. Not important. We should def make it private tho